src/components/dashboard/SidebarContent.js

```diff
@@ -34,6 +40,9 @@ class SidebarContent extends Component {
                     </SidebarSelector>}
                 </div>
                 <div className="flex h4 justify-between ph3 pv4 bt ">
+                     {profileImage != null &&
+                        <img src={profileImage} className="br-100 h3 w3 dib" alt="" />
```

Comment:
this should be passed from the Dashboard component rather than loaded from redux, we already have the user object there.